pkgs/development/haskell-modules/configuration-common.nix

```
@@ -2909,4 +2909,12 @@ self: super: {
     #   repa-query, repa-scalar, repa-store, repa-stream
   ;
 
+  postgresql-libpq = overrideCabal (drv: {
+    # Using use-pkg-config flag, because pg_config won't work when cross-compiling.
+    configureFlags = [ "-fuse-pkg-config" ];
```

Make sure it is still possible to override this in previous overlays.

```suggestion
    configureFlags = drv.configureFlags or [] ++ [ "-fuse-pkg-config" ];
```